lib/rucio/web/ui/flask/common/utils.py

```diff
@@ -360,8 +360,12 @@ def x509token_auth(data=None):
         msg = "<br><br>Your certificate (%s) is not mapped to (possibly any) rucio account: %s at VO: %s." % (html_escape(dn), html_escape(ui_account), html_escape(ui_vo))
     else:
         msg = "<br><br>Your certificate (%s) is not mapped to (possibly any) rucio account: %s." % (html_escape(dn), html_escape(ui_account))
-    msg += "<br><br><font color=\"red\">First, please make sure it is correctly registered in <a href=\"https://voms2.cern.ch:8443/voms/atlas\">VOMS</a> and be patient until it has been fully propagated through the system.</font>"
-    msg += "<br><br>Then, if it is still not working please contact <a href=\"mailto:atlas-adc-ddm-support@cern.ch\">DDM Support</a>."
+        if "None" in config_get("client", "ca_cert"):
```

Comment:
I did this on the basis that only the client was being passed a `CA_Certificate` location ...I can do it on any other basis though but this is was evident :)